airbyte-integrations/connectors/source-finn-statistics/source_finn_statistics/schemas/ads.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "clicks": {
+      "type": ["null", "integer"]
+    },
+    "emailNotifications": {
+      "type": ["null", "integer"]
+    },
+    "messagesReceived": {
+      "type": ["null", "integer"]
+    },
+    "favoriteMarkings": {
+      "type": ["null", "integer"]
+    },
+    "pushNotifications": {
+      "type": ["null", "integer"]
+    },
+    "lastChanged": { 
```

Comment:
I assume it's a datetime field. The following structure should work better during the parsing process
```
"lastChanged": {
  "type": ["null", "string"]
  "format": "date-time",
  "airbyte_type": "timestamp_without_timezone"
},
```

Reply:
Agree